frank/geometry.py

```diff
@@ -97,16 +104,15 @@ def deproject(u, v, inc, PA, inverse=False):
     sin_t = np.sin(PA)
 
     if inverse:
-        sin_t *= -1
+        u /= np.cos(inc)
+        up = u * cos_t + v * sin_t
+        vp = -u * sin_t + v * cos_t
+        return up, vp
```

Comment:
```suggestion
        sin_t *= -1
        u /= np.cos(inc)
```
The minimum changes needed (re-uses as much code as possible)

Reply:
Done in 85bb627

> Hi Jeff, well spotted.
> 
> I've requested a few small changes to geometry.py to keep the old code where possible because fewer changes are generally better. I'm fine with the rest of the changes in geometry.py. I'm not sure why the private function _fit_gemeotry_gaussian needed changing, but the code is no worse so I'm happy to leave it.
> 
> However, the changes to radial_fitters.py are wrong and need to be rolled back. The code originally computed the deprojected and phase corrected visibilities, so that if the source was off centre in the original data it will also be in the predicted data, as desired. I also don't think this code alters the u and v points passed in. It creates _new_ variables, u and v, that are the deprojected variables. This essentially "hides" the original u and v variables in the function, not overwriting them. The original u v and points (consistent with the CASA vis) should be unchanged.
> 
> Similarly, this means that the changes in io.py should be rolled back too.

Great, thanks for reviewing so quickly. I made all your changes in `geometry.py`. I also reverted my changes in `radial_fitters.py` and `io.py` (I don't think they weren't wrong, in that they gave the same result as the version in master, but they were thus redundant).

Reply:
Ok, I agree just redundant